coordinatorlib/src/main/scala/com/socrata/datacoordinator/secondary/SecondaryReplicationMessages.scala

```diff
@@ -0,0 +1,50 @@
+package com.socrata.datacoordinator.secondary
```

Comment:
my concept what to put it with the other provider things (since it is added to the Universe)